seed.js

```diff
@@ -6,37 +6,175 @@ process.env.OUTPUT_LOG = './log.txt';
 
 // create a password
 var crypto = require('crypto');
-var password = ''
-crypto.randomBytes(16).forEach(value => {
-  password += (value % 16).toString(16);
-})
-
 var app = require('express')();
 
 // stub out the schema
 require('./app/helpers/database')(app);
 
 // create an admin user
-var access = require('./app/helpers/access');
 var User = require('./app/models/user');
+var Game = require('./app/models/game');
+var Group = require('./app/models/group');
+var uuid = require('uuid/v1');
+var Release = require('./app/models/release');
 
-console.log('Creating admin user...');
-User.createUser(
-  {
+async function makeDummyData(){
+  var admin = await makeAdmin();
+  var game = await makeGame();
+  var groups = await makeGroups();
+  return [
+    admin,
+    game,
+    groups
+  ];
+}
+
+async function makeAdmin() {
+  console.log('Creating admin user...');
+  var passwordAdmin = '';
+  crypto.randomBytes(16).forEach(value => {
+    passwordAdmin += (value % 16).toString(16);
+  });
+  var adminGroup = new Group({
+    name: 'Admin2',
+    slug: 'Admin2',
+    token: 'myAccessToken',
+    tokenExpires: null,
+    privilege: 2,
+    isUserGroup: true
+  });
+  adminGroup.save();
+  var admin = new User({
     username: 'admin',
-    password: password,
+    password: passwordAdmin,
     email: 'empty@email.com',
-    name: 'Admin'
-  }, 
-  access.privilege.admin,
-  function(err, user)
-  {
-    if (err) {
-      throw err;
-      process.exit(1)
-    }
-
-    console.log(`User admin created with password ${password}`);
-    process.exit(0);
+    name: 'Admin',
+    groups: [adminGroup._id]
+  });
+  console.log(`User admin created with password ${passwordAdmin}`);
+  return admin.save();
+}
+
+async function makeGame(){
+  console.log('Creating example game with releases...');
+  //set your values as desired
+  var gameTitle = '';
```

Comment:
Is there any to declaring these variables? Could would we not just set the properties in the object to  be empty string?

  var gameParams = {
    title: '',
    slug: '',
    repository: '',
    location: '',
    description: 'A placeholder game for testing.'
  };

Reply:
Also we should avoid using var. I see no reason why we can't use const